release_docs/INSTALL_CMake.txt

```diff
@@ -1042,6 +1043,99 @@ The HDF5 CMake variables;
     HDF5_USE_PREGEN: set this to true
     HDF5_USE_PREGEN_DIR: set this path to the preset H5Tinit.c file
 
+
+========================================================================
+X:    Using CMakePresets.json for compiling
+========================================================================
+
+One problem that CMake users often face is sharing settings with other people for common
+ways to configure a project. This may be done to support CI builds, or for users who
+frequently use the same build. CMake supports two main files, CMakePresets.json and CMakeUserPresets.json,
+that allow users to specify common configure options and share them with others. CMake also supports
+files included with the include field.
+
+CMakePresets.json and CMakeUserPresets.json live in the project's root directory. They
+both have exactly the same format, and both are optional (though at least one must be
+present if --preset is specified). CMakePresets.json is meant to specify project-wide build
+details, while CMakeUserPresets.json is meant for developers to specify their own local build details.
+
+See CMake documentation for details: https://cmake.org/cmake/help/latest/manual/cmake-presets.7.html
+
+HDF-provided CMakePresets.json
+-------------------------------
+The CMakePresets.json provided by HDF requires CMake version 3.25, which supports package
+and workflow presets, and ninja build system. The top-level configuration group is intended to be
+a standard set of options to produce a package of shared and staic libraries and tools. Other configurations
+used for inheriting settings are in the included json file in "config/cmake-presets/hidden-presets.json".
+
+Available configurations presets can be displayed by executing:
+    cmake -S <path-to-source> --list-presets
+
+Using individual command presets (where <compiler-type> is  GNUC or MSVC or Clang):
+    change directory to the hdf5 source folder
+    cmake --presets=ci-StdShar-<compiler-type>
+    cmake --build --presets=ci-StdShar-<compiler-type>
+    ctest --presets=ci-StdShar-<compiler-type>
+    cpack --presets=ci-StdShar-<compiler-type>
+
+
+Using the workflow preset to configure, build, test and package the standard configuration is:
+    change directory to the hdf5 source folder
+    execute "cmake --workflow --presets=ci-StdShar-<compiler-type> --fresh"
+          where <compiler-type> is  GNUC or MSVC or Clang
+
+Creating your own configurations
+--------------------------------
+The quickest way is to copy CMakePresets.json to CMakeUserPresets.json and
+edit CMakeUserPresets.json configuration names from ci-* to my-*. Change the
+configurePresets inherits only for those that you have provided alternates for
```

Comment:
This sentence reads a bit weird to me

Reply:
The explanation is more complicated then the process - Just a first attempt, open for suggestions.

Reply:
I think the other explanations here are fine. I'm just not sure what "Change the
configurePresets inherits only for" means here. What is configurePresets? Does "inherits" here refer to the inherits configuration setting in the file?

Reply:
This refers to : 
 "packagePresets": [
    {
      "name": "ci-StdShar-MSVC",
      "configurePreset": "ci-StdShar-MSVC",
      "inherits": "ci-x64-Release-MSVC"
    },

Reply:
And the actual configurePresets section which has "inherits"
i.e.
 "packagePresets": [
    {
      "name": "ci-StdShar-MSVC",
      "configurePreset": "ci-StdShar-MSVC",
      "inherits": "ci-x64-Release-MSVC"
    },

Reply:
"my-base-tgz" changed from "ci-base-tgz"

Reply:
I could have just included my User file in the docs folder maybe?

Reply:
I think wrapping inherits in quotes and maybe changing to 'the "inherits" field' would do a lot to clarify that it refers to editing the "inherits" field in the configuration file. As for "configurePresets" maybe adding "entries" after that word or showing an inline example would make it clearer.

Reply:
Wrapped the presets names in quotes